### PR TITLE
fix(blog): MagMutual - brand names

### PR DESCRIPTION
**Pull request by muescha**
## Description

changed:

- brandname from `GatsbyjS` to `Gatsby
- brandname from magmutual.com to MagMutual.com (like other MagMutual.com on the page)
- fixed link from `http` to `https`

## Related Issues

- #22602 `Blog case study mag mutual`

### CODE REVIEW

File: docs/blog/2020-03-30-Fast-Secure-Flexible-MagMutual/index.md [hunk 1]
```diff
@@ -98,5 +98,5 @@ MagMutual serves the intersection of two rapidly evolving industries: insurance
 ## Resources
 
 - See the [complete case study for MagMutual.com](https://www.mediacurrent.com/work/case-study/magmutual-drupal-8-gatsby).
-- Learn more about some of the development tools used for this project in our webinar, [Rain + GatsbyJS: Fast-Tracking to Drupal 8](https://www.mediacurrent.com/videos/webinar-recording-rain-gatsbyjs-fast-tracking-drupal-8).
+- Learn more about some of the development tools used for this project in our webinar, [Rain + Gatsby: Fast-Tracking to Drupal 8](https://www.mediacurrent.com/videos/webinar-recording-rain-gatsbyjs-fast-tracking-drupal-8).
```
Comment (muescha) [inline docs/blog/2020-03-30-Fast-Secure-Flexible-MagMutual/index.md:101]: unsure with this one, because it is used on the linked page as title 
🤔

**Comment by AishaBlake** [comment]
Thanks as always for the contribution, @muescha! I'm going to close this PR because I wouldn't want to make most of these changes without the knowledge/consent of the author. 

The question of capitalization, for example, is a bit subjective in this case. Generally, I look for only very clear typos when deciding whether to update blog posts.

**Comment by muescha** [comment]
ok - should then we ping to @mgienow ?

**Comment by muescha** [comment]
@AishaBlake can i ask for a new review 


i done my changes because of:

- magmutal case: i make the case consistent over the blog page. because there are on this page:
  - 4x MagMutual.com
  - 1x magmutual.com

- Gatsby**js** instead of Gatsby is slipped into text while rewording section `Mediacurrent’s Solution` from the original blog post (where also only used `Gatsby`) on https://www.drupal.org/case-study/magmutual-insurance-drupal-8-gatsby-redesign

- i was only unsure with the wrong brand name spelling in link in the section `Resources`, because that is the title of the linked post.

**Comment by mgienow** [comment]
These changes look reasonable to me. Especially good catch on changing GatsbyJS to Gatsby, and even if the link text in our doc is different from the title on the target page, I don't see that as a problem. I can also ping Aten to fix it on their page.

**Comment by muescha** [comment]
because of answer from @mgienow i reopened the PR and ask for a new review :)

**Comment by laurieontech** [comment]
Thanks for chiming in @mgienow!

**Pull request merged**